### PR TITLE
please review & merge

### PR DESCRIPTION
**Pull request by dortegadelv**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

### CODE REVIEW

**Comment by dortegadelv** [comment]
I want to upload our program PReFerSim into the Bioconda for reproducibility purposes. Link to the paper describing our program:
https://academic.oup.com/bioinformatics/article/32/22/3516/2525599

**Comment by dpryan79** [comment]
Does this not need any of the text files in the git repo?

File: recipes/PReFerSim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: prefersim
+  version: '1.0'
+
+source:
+  url: https://github.com/LohmuellerLab/PReFerSim
```
Comment (dpryan79) [inline recipes/PReFerSim/meta.yaml]: You need to tag a release

Comment (dortegadelv) [reply recipes/PReFerSim/meta.yaml]: I think I solved this. Just tagged a release.

The text files contain some examples to run the code and are included in the release. They can be helpful for the user to run their analysis. I would like to keep them in the release folder.

Thanks for your help!

**Pull request merged**